framework/src/modules/governance/events/proposal_created.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BaseEvent } from '../../base_event';
+import { ProposalType } from '../types';
+
+interface ProposalCreatedEventData {
+	creator: Buffer;
+	index: number;
+	type: ProposalType;
+}
+
+export class ProposalCreatedEvent extends BaseEvent<ProposalCreatedEventData> {}
```

Comment:
schema is missing?

Reply:
I was planning to add the event schema and event logging as part of event quest #8869